src/library/dao/cuedao.cpp

```diff
@@ -221,18 +219,16 @@ void CueDAO::saveTrackCues(
         cueIds.append(pCue->getId().toString());
     }
 
-    // Delete orphaned cues
+    // Purge orphaned cues
```

Comment:
Please avoid to change code, formatting, or comments based on personal preferences without improving anything. This just causes noise.

Reply:
I am pretty sure we use the term "purge" whenever something gets removed irrevocably, whereas "delete" is a more broad term. So it seemed sensible to make it more precise here.

Reply:
Purge is used in the UI. It doesn't make sense to rephrase the following SQL "DELETE"  command in the comment to "purge". Why should we use 2 different verbs in the same context??

Reply:
The SQL command is rather general - for machine-readabilty. Aren't the comments supposed to be as specific and clear as possible (without sacrificing conciseness)?

Reply:
I don't see any value and am not going to discuss this again.